update/update-version.md

```diff
@@ -1,139 +1,187 @@
+1. Get the VPC IDs and the regions of any VPCs that the VSIs are deployed in by using the IBM Cloud console:
+    1. Click the 3-layer menu icon, and then click **VPC Infrastructure** > **VPCs**.
+    1. Select the region in which the VPC of the VSI was deployed.
+    1. Select the VPC that is associated with the VSI deployment.
+    1. Copy the **VPC ID**.
+1. Run the script:
 
-1. Run the script from your local machine
     ```sh
     bash schematics_update_v3.x.x_to_v4.x.x.sh -v "<vpc-id1>[,<vpc-id2>,...]" -r "<vpc-region>" [-k "<vpc-ibm-api-key>"]
     ```
-    This script will trigger a new job in the schematics workspace, please monitor the state of the job in IBM Cloud UI. Check if the job completes **Successfully**, in that case go ahead with **Step 7**. In case of schematics workspace job ends in a failed state, please follow **Step 9** to revert any changes made by the script.
 
-1. The user would have to update their code to consume version 4.x.x, and then update their schematics workspace to the version of their code that contains the updated module and click on `Generate plan` and make sure none of the VSIs will be recreated. The expected behaviour should be that the plans contains only naming updates, and no destroy / re-creates of any resources.
+    - Replace `<vpc-id1>` and `<vpc-region>` with the information that you copied earlier.
+    - If the VSIs are deployed in an account that is different from the account where the Schematics workspace is located, include the `-k` option and replace `<vpc-ibm-api-key>` with the IBM Cloud API key with access to the VPC.
 
-1. Click on `Apply plan`.
+    The script creates a job in the Schematics workspace.
 
-1. [Optional] If a schematics workspace job that was initiated through the script encounters an issue, you can undo any modifications made by running the script again with the -z flag. For example,
-    ```sh
-    bash schematics_update_v3.x.x_to_v4.x.x.sh -z
-    ```
-    A new schematics workspace job reverting the state back to its prior condition, which existed prior to the execution of the script. The script will use the `revert.json` file that got created when the script initially ran
+1.  Monitor the status of the job in the IBM Cloud [console](https://cloud.ibm.com/schematics/workspaces).
+    - When the job completes successfully, go to the next step.
+    - If the job fails, see [Reverting changes](#reverting-changes).
 
-1. After the successful upgrade to the newer release of the VSI module, it is safe to remove the temporary files generated by the script.
-    ```sh
-    rm moved.json revert.json
-    ```
+### Apply the changes in Schematics
+
+1. Update your code to consume version 4.x.x, and then update your Schematics workspace to the version of the code that contains the updated module. Click **Generate plan** and make sure none of the VSIs will be re-created. You should see changes only to names. No resources should be set to be destroyed or re-created.
```

Comment:
> You should see changes only to names

^ Perhaps use terraform terminology and say "You should see update in place changes only"